code/lib/cli/src/initiate.ts

```diff
@@ -320,16 +323,10 @@ async function doInitiate(options: CommandOptions, pkg: PackageJson): Promise<vo
 
     // Add a new line for the clear visibility.
     logger.log();
-    return;
+    throw new HandledError(`Angular project already installed`);
   }
 
-  const installResult = await installStorybook(
-    projectType as ProjectType,
-    packageManager,
-    options
-  ).catch((e) => {
-    process.exit();
-  });
+  const installResult = await installStorybook(projectType as ProjectType, packageManager, options);
```

Comment:
@tmeasday This is new behavior right? Since it's not throwing a handled error. But this seems like a good change